cmd/rendmp/src/lib.rs

```diff
@@ -1520,20 +1553,49 @@ fn rendmp_phase_check<'a>(
         bail!("must be in A2 when checking phases");
     }
 
+    if cmd_spd::spd_any(hubris, core)? {
```

Comment:
Nit: In my Humility reorganization, I've been trying to avoid command-to-command dependencies, since that introduces a longer dependency chain when building.  IMHO the Right Way™ to do this would be to add a new crate (`humility-spd`), then use it both here and in `cmd-spd`.

I don't feel strongly enough about this to hold up the PR, though.

Reply:
I'm going to split the difference on this and add a comment to make clear that this is not a practice to be emulated!